src/PhpSpec/Matcher/ThrowMatcher.php

```diff
@@ -261,15 +280,21 @@ private function getException(array $arguments)
             return $arguments[0];
         }
 
-        if (is_object($arguments[0]) && $arguments[0] instanceof \Exception) {
-            return $arguments[0];
+        if (is_object($arguments[0])) {
+            if (PHP_VERSION_ID >= 70000 && $arguments[0] instanceof \Throwable) {
```

Comment:
I'm not sure how this will work with the various HHVM version numberings. Might it be better to `class_exists` on the `Throwable` type?